docs/cluster_up_down.md

```diff
@@ -144,24 +151,28 @@ To stop your cluster, run:
 $ oc cluster down
 ```
 
-### Mac OS X with Docker Toolbox
+### Mac OS X with Docker Machine
 
-1. Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox) and ensure that it is functional.
-2. Install the oc binary using homebrew with: `brew install openshift-cli`
+OpenShift is currently compatible with version 17.09.1 of Docker for Mac. 
+
+1. Install [Docker for Mac](https://docs.docker.com/docker-for-mac/release-notes/) making sure you meet the [prerequisites](https://docs.docker.com/docker-for-mac/#/what-to-know-before-you-install) 
+and download a compatible version of Docker.
+2. Install [VirtualBox for OSX Hosts](https://www.virtualbox.org/wiki/Downloads)
+3. Install the oc binary using homebrew with: `brew install openshift-cli`
 
    OR
 
    Download the OS X `oc` binary from [openshift-origin-client-tools-VERSION-mac.zip](https://github.com/openshift/origin/releases) and place it in your path.
 
    > Please be aware that the 'oc cluster' set of commands are only available in the 1.3+ or newer releases.
 
-3. Open Terminal and run
+4. Open Terminal and run
```

Comment:
This option is getting removed. A better approach is to create the docker machine yourself, then set its environment (see instructions here: https://docs.docker.com/machine/get-started/#run-containers-and-experiment-with-machine-commands). Once that's setup, simply run `oc cluster up` as normal.